### PR TITLE
Feature/pxweb2 286 footer link location test scb

### PR DESCRIPTION
**Pull request by KentMossback**
Change link-location in footer for project information. Updated with latest changes from master

### CODE REVIEW

File: apps/pxweb2/src/app/app.tsx [hunk 1]
```diff
@@ -461,30 +472,50 @@ export function App() {
...
-                selectedValues={selectedVBValues}
-                onChangeCodeList={handleCodeListChange}
-                onChangeMixedCheckbox={handleMixedCheckboxChange}
-                onChangeCheckbox={handleCheckboxChange}
-              />
-            )
-        )}
-    </div>
+    <>
+      <select
```
Comment (SjurSutterudSagen) [inline apps/pxweb2/src/app/app.tsx:476]: The test site should not have this Select for different tables

**Review by SjurSutterudSagen — CHANGES_REQUESTED** [review]

File: apps/pxweb2/src/app/app.tsx [hunk 2]
```diff
@@ -484,7 +495,7 @@ export function App() {
               />
             )
         )}
-    </div>
+    </div>
```
Comment (SjurSutterudSagen) [inline apps/pxweb2/src/app/app.tsx:498]: We should remove these empty spaces here.

**Review by SjurSutterudSagen — CHANGES_REQUESTED** [review]

File: apps/pxweb2/src/app/app.tsx (hunk 2 above)
Comment (KentMossback) [reply apps/pxweb2/src/app/app.tsx]: Removed as suggested

**Review by SjurSutterudSagen — APPROVED** [review]
Looks good

**Pull request merged**